_docs/_latest/setup/upgrade.md

```diff
@@ -151,6 +158,173 @@ change a certain resource, you'll need to create a new deployment of Forseti.
 
 Learn more about [Updating a Deployment](https://cloud.google.com/deployment-manager/docs/deployments/updating-deployments).
 
+{% capture upgrading_2_0_0_to_2_1_0 %}
+
+1. Open cloud shell when you are in the Forseti project on GCP.
+1. Checkout forseti with tag v2.1.0 by running the following commands:
+    1. If you already have the forseti-security folder under your cloud shell directory, 
+    run command `rm -rf forseti-security` to delete the folder.
+    1. Run command `git clone` to clone the forseti-security directory to cloud shell.
```

Comment:
Do you feel we should give the full instruction here?  It seems odd that this is abbreviated here, but the full commands are given elsewhere.